### ISSUE TITLE
Responsive/smaller image for picture of you

### ISSUE THREAD

**Issue by MarauderXtreme** [JustusAdam/justusadam.github.io#14]
While providing a high resolution pictures is a nice addition to your page it slows it noticeably down.  
Or lets put it that way. The image load takes a second while everything else is already present. 1.4MB of 1.8MB is a sizeable footprint for just one image and it kills mobile data.

Proposal:

Reduce the resolution of the image to 336x252 pixel. (That's the max size I could find for your current styling) and maybe keep the full resolution image as a link.

You could also use a [responsive approach][0] with `srcset`s. That is the better approach in my eyes and you could serve even more appropriate images for certain devices.

[0]: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images

Sneak Edit: Fix Markdown

**Comment by JustusAdam** [comment]
Yeah cool idea. I just don't know when I'll be having the time to do this. 😄
Btw did you notice that I added an online CV at https://justus.science/cv? I've had no one proof read that yet so if you want to point out some embarrassing mistakes to me that's probably the place to do it 😄.

**Comment by MarauderXtreme** [comment]
I've seen the fork. When I find time I will coldbloodedly judge you and your alleged mistakes.

You could enable Issues for it for a clearer separation?

**Comment by JustusAdam** [comment]
I wasn't aware the issues were closed :sweat_smile:

**Comment by MarauderXtreme** [comment]
When you fork a repo they normally are disabled. I am assuming that a fork is seen as temporary for contributions to the main repository so there is no need for issues.
